src/hotspot/share/code/pcDesc.cpp

```diff
@@ -43,17 +43,12 @@ address PcDesc::real_pc(const CompiledMethod* code) const {
 void PcDesc::print_on(outputStream* st, CompiledMethod* code) {
 #ifndef PRODUCT
   ResourceMark rm;
-  st->print("PcDesc(pc=" PTR_FORMAT " offset=%x bits=%x):", p2i(real_pc(code)), pc_offset(), _flags);
+  st->print_cr("PcDesc(pc=" PTR_FORMAT " offset=%x bits=%x):", p2i(real_pc(code)), pc_offset(), _flags);
```

Comment:
This changes the output from
```
PcDesc(pc=0x00007fe00d12475f offset=ffffffff bits=0):
PcDesc(pc=0x00007fe00d12478c offset=2c bits=0):   Test::test3@-1 (line 33)
``` 
to
```
PcDesc(pc=0x00007f4fc9035a5f offset=ffffffff bits=0):
PcDesc(pc=0x00007f4fc9035a8c offset=2c bits=0):
   Test::test3@-1 (line 33)
``` 
But since that is more inline with the output of `ScopeDesc::print_on`, that's fine with me.